build_html.sh

```diff
@@ -3,6 +3,10 @@
 # build_html.sh
 # build website based on jupyter notebooks
 
+echo "Try to fetch daily backup"
+date=$(date '+%Y-%m-%d')
+aws s3 sync s3://d2l-java-notebook/daily-backup/$date .
```

Comment:
have you test this locally? you need to sync it to the $output_dir : d2l-java/_build/eval/